sqlgen/db.go

```diff
@@ -204,7 +206,7 @@ func (db *DB) checkColumnValuesAgainstLimit(columns []string, values []interface
 	return nil
 }
 
-func (db *DB) checkColumnValuesAgainstLimits(ctx context.Context, columns []string, values []interface{}, tableName string) error {
+func (db *DB) checkColumnValuesAgainstLimits(ctx context.Context, columns []string, values []interface{}, tableName string, query SQLQuery) error {
```

Comment:
tiny nit, but could we put the query as the first param after ctx? query, columns, tablename

Reply:
Done